lib/src/utils/yaml_utls.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:yaml/yaml.dart';
```

Comment:
Typo in file name. yaml_utls.dart => yaml_ut**i**ls.dart

Reply:
fixed